### PR TITLE
Add url to NuGet package

### PR DESCRIPTION
**Pull request by bjarnef**
Issue: http://issues.umbraco.org/issue/OUR-435

I have added a property to enter url to NuGet package, instead of people enter these in package description and in different way.

image

image

### CODE REVIEW

File: OurUmbraco.Client/src/scss/elements/_package-detail.scss [hunk 1]
```diff
@@ -79,6 +79,53 @@
...
+			font-family: Consolas, Menlo, Monaco, Lucida Console, Liberation Mono, DejaVu Sans Mono, Bitstream Vera Sans Mono, Courier New, monospace, serif;
+			font-size: 0.8rem;
+			transform: translate(15px, -46%);
+			opacity: .9;
+		}
+        .pre {
+            padding-left: 45px;
+        }
+	}
+
```
Comment (bjarnef) [inline OurUmbraco.Client/src/scss/elements/_package-detail.scss]: This is more or less the same styles as for Install-Package UmbracoCms on download page. Maybe the default style of .pre can be moved to one of the main .scss files.

**Comment by nul800sebastiaan** [comment]
Awesome! I added some code to make the extra field on the document type: https://github.com/umbraco/OurUmbraco/commit/a81a3556f9f13d522fcf56e7b1d4c862f7958a25

**Comment by bjarnef** [comment]
Great! 👍 
Thanks for looking at this 🥇

**Comment by nul800sebastiaan** [comment]
No problem, thanks for updating after all those changes I made today! 😁 Changes are rolled out now!

**Comment by bjarnef** [comment]
Thanks, it works! :smile: https://our.umbraco.org/projects/backoffice-extensions/urlpicker/

**Pull request merged**